src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -188,9 +188,7 @@ public async Task DownloadQualityProfile_Success()
                 .ReturnsAsync(bindingConfig);
 
             var bindingArgs = CreateBindCommandArgs("key", ProjectName, new ConnectionInformation(new Uri("http://connected")));
-            var testSubject = this.CreateTestSubject(bindingArgs, configProviderMock.Object);
-
-            host.SupportedPluginLanguages.Add(language);
+            var testSubject = this.CreateTestSubject(bindingArgs, configProviderMock.Object, languagesToBind: new[] { language });
```

Comment:
Fixed up existing tests (most of the tests only expected a few binding languages).